x11-libs/libice/libice-1.0.9.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Inter-Client Exchange Library"
+DESCRIPTION="The ICE library utilizes callbacks to process incoming \
+messages. Using callbacks allows ProtocolSetup messages and authentication \
+to happen behind the scenes. An additional benefit is that messages never \
+need to be buffered up by the library when the client blocks waiting for \
+a particular message."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
+COPYRIGHT="1993, 1998 The Open Group"
+LICENSE="MIT (no promotion)"
+REVISION="1"
+SOURCE_URI="https://www.x.org/releases/individual/lib/libICE-$portVersion.tar.gz"
+CHECKSUM_SHA256="7812a824a66dd654c830d21982749b3b563d9c2dfe0b88b203cefc14a891edc0"
+SOURCE_FILENAME="libICE-$portVersion.tar.gz"
+SOURCE_DIR="libICE-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=6.3.0
+libVersionCompat="$libVersion compat >= 6"
```

Comment:
I suggest using this construct as this save any future maintainer from editing an another line, which is easily forgotten (I did once and haikuporter doesn't catch it...)

```diff
-libVersionCompat="$libVersion compat >= 6"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```

Reply:
Yeah, only seen it recently, so I need to get a hang of that one :)

Reply:
That's easy to remember:
* \# is for doing pattern maching starting from the left (hint: left = beginning of the line = where you put \# for comments)
* % is for doing pattern maching starting from the right (hint: the percentage character is on the right of the figures)
* use \#\# or %% to make the size of the pattern as big as possible.